snapcraft.yaml

```diff
@@ -47,11 +54,14 @@ apps:
   rax2:
     command: usr/bin/rax2
     plugs: [home]
+  rasign2:
+    command: usr/bin/rasign2
+    plugs: [home]
 
 parts:
   radare2:
-    source: https://github.com/radareorg/radare2.git
-    source-tag: '2.6.0'
+    source: https://github.com/radare/radare2.git
```

Comment:
No, it's actually `https://github.com/radareorg/radare2.git`

Reply:
I changed the URL and added the documentation.